### ISSUE TITLE
Accessibility: WCAG 3.2.5 Change on Request Level AAA

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyone#83]
Changes of context are initiated only by user request or a mechanism is available to turn off such changes.
Reference:
https://www.w3.org/TR/WCAG21/#change-on-request
https://www.w3.org/WAI/WCAG21/quickref/#change-on-request

Core widgets uses select lists that unexpectedly change and will fail this guideline.

-When a select list is expanded by pressing enter or space, and tab is pressed next, the item is opened and the page is refreshed.
-When a select list is **in focus**  but not expanded and an arrow key is used, the item is opened and the page is refreshed. 
Meaning a user can't even read the items in the list because moving up and down the list opens the item directly.

 @audrasjb I was not able to find a trac ticket for this. I would like to ask the accessibility team if this is:
-A non issue, in case I misunderstood the guideline and how users use these select lists.
-Something that the theme should solve.
-Something that the theme does not need to solve for 5.6 because it is a core problem. 

-But because this is a level AAA requirement core is not likely to solve it because it only requires level AA?
